src/main/java/org/codeforamerica/messaging/models/Template.java

```diff
@@ -0,0 +1,66 @@
+package org.codeforamerica.messaging.models;
+
+import com.github.jknack.handlebars.Handlebars;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.util.Map;
+import java.util.function.Function;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Template {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+    @NotBlank
+    String name;
+    String subject;
```

Comment:
I missed this in the email sending story. It seems reasonable to specify a length limit here.
> We recommend you use no more than 9 words and 60 characters. [Best Practices for Email Subject Lines](https://mailchimp.com/help/best-practices-for-email-subject-lines/)

(adding to story)

Reply:
I'm not sure if it's a good idea to try to limit the size of the subject template, unless you imagining that the subject is static? Maybe we can add warnings about emails/templates being created with long titles as part of the validation?

Reply:
I was but I see your point. Warnings if the subject has no placeholders or if reasonable placeholder substitutions would exceed norms.